scripts/build/Platform/Windows/build_config.json

```diff
@@ -45,7 +46,7 @@
       "SCRIPT_PARAMETERS": "--platform=Windows --repository=%REPOSITORY_NAME% --jobname=%JOB_NAME% --jobnumber=%BUILD_NUMBER% --jobnode=%NODE_LABEL% --changelist=%CHANGE_ID%"
     }
   },
-  "test_impact_analysis_profile": {
+  "test_impact_analysis_profile_native": {
```

Comment:
Just confirming: Do we intend to run this in AR? It's not on any pipeline steps or contain a suffix in the name to allow it to run nightly.

Reply:
Yes. We are not intending to run `test_impact_analysis_native` in AR at this time.